job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -183,6 +186,51 @@ BEGIN
          ELSE quote_ident(in_sort_field)
        END ||
         ' ' || CASE WHEN in_sort_ascending THEN 'ASC' ELSE 'DESC' END;
-    RETURN QUERY EXECUTE sql;
+
+    -- Create temporary table as a base to update the job progress
+    EXECUTE 'CREATE TEMPORARY TABLE get_job_temp ON COMMIT DROP AS ' || sql;
+
+    ALTER TABLE get_job_temp ADD COLUMN id SERIAL PRIMARY KEY;
+
+    -- Create an array of job_id(s) based on the get_job_temp table
+    jobArray := ARRAY(SELECT DISTINCT (jt.job_id) FROM get_job_temp jt);
+
+    -- Check that the array is not empty
+    IF array_length(jobArray, 1) > 0 THEN
```

Comment:
Do you actually need to check if the array is empty?  Wouldn't it just go around the loop zero times?

Reply:
It seems so, I ran the tests again without it and getting
![image](https://user-images.githubusercontent.com/1658429/94148346-6a35f480-fe6e-11ea-93c6-7dddbaf30d20.png)

Reply:
Ok.